chess-dataviz-1.0.0/README.md

```diff
@@ -0,0 +1,25 @@
+chess-dataviz
+=============
+
+chess-dataviz is a visualization library for chess, written for D3. It has a number of visualizations to show statistics on games, such as:
```

Comment:
@bennyrubanov there are a lot of unrelated changes in this PR, did you intend to add these? Would prefer to separate the review. Looks like you cloned the chess dataviz libary. Any reason we don't just want to use it as a package/import?

Reply:
answered in discord